bin/node/runtime/pangolin/src/pallets/ethereum_issuing.rs

```diff
@@ -8,10 +8,15 @@ frame_support::parameter_types! {
 	pub const EthereumIssuingPalletId: PalletId = PalletId(*b"da/ethis");
 }
 
+frame_support::parameter_types! {
+	pub RawCallGasLimit: U256 = U256::from(300_000_000);
```

Comment:
there are so many these same definitions. Can it be defined at a common place?

Reply:
Initially, I want to make this gas value configurable. It is not necessary to do so at the moment.

Reply:
Fixed.